tests/nativeapp/test_post_deploy_for_app.py

```diff
@@ -102,69 +111,15 @@ def test_sql_scripts(
         [
             mock.call(f"use database {app_model.fqn.name}"),
             mock.call(f"use database {app_model.fqn.name}"),
-        ]
+        ],
+        any_order=True,
     )
     assert mock_execute_queries.mock_calls == [
-        mock.call(post_deploy_1),
-        mock.call(post_deploy_2),
+        mock.call(DEFAULT_POST_DEPLOY_CONTENT_1),
+        mock.call(DEFAULT_POST_DEPLOY_CONTENT_2),
     ]
 
 
-@mock.patch(SQL_EXECUTOR_EXECUTE)
-@mock.patch(SQL_EXECUTOR_EXECUTE_QUERIES)
-@mock.patch(CLI_GLOBAL_TEMPLATE_CONTEXT, new_callable=mock.PropertyMock)
-@mock_connection()
-@mock.patch(MOCK_CONNECTION_DB, new_callable=mock.PropertyMock)
-@mock.patch(MOCK_CONNECTION_WH, new_callable=mock.PropertyMock)
-@mock.patch.dict(os.environ, {"USER": "test_user"})
-def test_sql_scripts_with_no_warehouse_no_database(
```

Comment:
I decided to pass the role/warehouse to the sql facade in execute_post_deploy_hooks so that it wasn't dependent on previous calls to use_role/warehouse by the executor, which makes this test invalid with the new setup